bip-0324/reference.py

```diff
@@ -566,7 +566,7 @@ def get_keystream_bytes(self, nbytes):
         return ret
 
     def crypt(self, chunk):
-        """Encrypt or decypt chunk."""
+        """Encrypt or decrypt chunk."""
```

Comment:
I was initially confused how this function does both encryption and decryption, but per the BIP text on FSChaCha20: "Encryption and decryption are identical here, so a single function <code>crypt</code> is exposed."